tests/tests.js

```diff
@@ -1,14 +1,42 @@
 const tape = require('tape');
-const request = require('supertest');
+const supertest = require('supertest');
 const server = require('../server.js');
+const request = require('request');
 
 tape('home route test: GET request to /', t => {
-  request(server)
+  supertest(server)
     .get('/')
     .end((err, res) => {
-      if (err) throw err;
+      t.error(err, 'No Erro');
       t.equal(res.status, 200, 'should return status code 200');
       t.ok(res.text.includes('<div id="list-page-content">'), 'should return home page containing test string "<div id="list-page-content">"');
       t.end();
     });
 });
+
+tape('Test the event details woute', t => {
+  const htmlSample = '<h1 class="text-warning eventName">default</h1>';
+  supertest(server)
+    .get('/events/59afaf75b0a5e80011beeafa')
```

Comment:
where is this id coming from?

Reply:
I added it manually and now thinking about it it's probably bad practice.
whats the better way to do it ?

Reply:
I don't know 😕

Reply:
you could query the API first to get all events, then use one of the id's you get back?

But this is a bit long

Reply:
yeah but that sounds better, i'll just make a GET to the api take the first object and use it to test